pkg/kudoctl/resources/dependencies/operatorversion.go

```diff
@@ -0,0 +1,27 @@
+package dependencies
+
+import (
+	"github.com/kudobuilder/kudo/pkg/apis/kudo/v1beta1"
+	engtask "github.com/kudobuilder/kudo/pkg/engine/task"
+)
+
+// UpdateKudoOperatorTaskPackageNames sets the 'Package' and 'OperatorName'
```

Comment:
that is some helper method that only marginally has something to do with the dependencies. not sure if a separate file within the `dependencies` package is the right place 🤔

Reply:
I agree, do you have an idea for a better place? I chose `dependencies` because it's updating fields related to dependencies.

Reply:
`operatorversion_types_helpers.go` maybe?

Reply:
IMHO, too specific for a broader `helpers` file. It is called in only one place: (`install/operators.go`) so maybe leave it a private method there?

Reply:
You're right, it's only called once! This used to be different in an older iteration of the refactoring. Makes sense to make it a private method again.